app/controllers/people/index.js

```diff
@@ -34,14 +36,26 @@ export default class PeopleIndexController extends Controller {
 
   get hasNoResults() {
     return (
-      this.model.loadPeopleTaskInstance.isFinished && this.people.length === 0
+      this.model?.loadPeopleTaskInstance?.isFinished &&
+      this?.people?.length === 0
     );
   }
 
   get hasErrored() {
     return this.model.loadPeopleTaskInstance.isError;
   }
 
+  @action
+  setPosition(event) {
```

Comment:
nitpick: the argument is the record, not an event. I would rename it to `position` since it is a bit confusing at the moment.